Pulumi.dev.yaml

```diff
@@ -0,0 +1,4 @@
+config:
+  gcp:project: chef-216716
+  gcp:region: us-central1
```

Comment:
why choosing a region in USA if you are in europe ?

Reply:
Because all example mostly  used region in the USA, but I must investigate this theme